packages/address/src/__tests__/MultisigAddress.spec.ts

```diff
@@ -0,0 +1,85 @@
+import Address from '..'
+import { MultisigAddress } from '../MultisigAddress'
+import {  
+  bobB58,
```

Comment:
There are a bunch of lint errors to fix here too. You can ignore any of the dependency cycle ones, we need a refactor to fix those but all others can be fixed.